forms/edit_profile.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from wtforms_alchemy import ModelForm
+
```

Comment:
unnecesary blank lines in `from ` import

Reply:
indeed